### PR TITLE
Contributing code via pull requests

### PR DESCRIPTION
**Pull request by roshnaeem**
**Description:**
Updating the [Contributing code via pull requests](https://arviz-devs.github.io/arviz/contributing/contributing_prs.html).

### CODE REVIEW

File: doc/source/contributing/contributing_prs.md [hunk 1]
```diff
@@ -82,34 +80,47 @@ Section in construction
 
 1. Fork the [project repository](https://github.com/arviz-devs/arviz/) by clicking on the 'Fork' button near the top right of the main repository page. This creates a copy of the code under your GitHub user account.
 
-2. Clone your fork of the ArviZ repo from your GitHub account to your local disk, and add the base repository as a remote:
+2. Clone your fork of the ArviZ repo from your GitHub account to your local disk.
```
[the remark this answers is not in the document]
Comment (roshnaeem) [reply doc/source/contributing/contributing_prs.md]: Done.

File: doc/source/contributing/contributing_prs.md [hunk 2]
```diff
@@ -82,34 +79,71 @@ Section in construction
...
+   ::::
 
-3. Create a ``feature`` branch to hold your development changes:
+4. Create a ``feature`` branch to hold your development changes:
 
    ```bash
    $ git checkout -b my-feature
    ```
 
    Always use a ``feature`` branch. It's good practice to never routinely work on the ``main`` branch of any repository.
```
Comment (OriolAbril) [inline doc/source/contributing/contributing_prs.md:134]: Seeing how we often get PRs from main, maybe we could put this two sentences inside an important or warning admonitions so it pops up? Now that we have this here and not on github we can use all sphinx powered cool formatting

Comment (roshnaeem) [reply doc/source/contributing/contributing_prs.md:134]: Sure

File: doc/source/contributing/contributing_prs.md [hunk 3]
```diff
@@ -82,34 +79,71 @@ Section in construction
...
+
+   ::::
+
+4. Create a ``feature`` branch to hold your development changes:
 
    ```bash
    $ git checkout -b my-feature
    ```
+  > :warning: Always create a new ``feature`` branch before making any changes. Make your chnages
+    in the ``feature`` branch. It's good practice to never routinely work on the ``main`` branch of any repository.
```
Comment (OriolAbril) [inline doc/source/contributing/contributing_prs.md:134]: this renders a bit strangely, it looks like a workaround to get admonitions in markdown. MyST supports them natively with the syntax shown in https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#directives-a-block-level-extension-point and https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions. 

Here is the list of all available admonitions for reference: https://sphinx-book-theme.readthedocs.io/en/latest/reference/kitchen-sink/paragraph-markup.html#admonitions

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**